style.css

```diff
@@ -0,0 +1,221 @@
+body{
+	width:70%;
+	 margin: auto;
+	 margin-top:50px;
+	 margin-bottom:50px;
+	 box-shadow: -100px 100px 200px -90px #8c8c8c;
+	 border:0.25px solid #DCDCDC;
+	 min-width:500px;
+}
+
+section{
+	 background-color: #f0f0f0; /*color!*/
+}
+
+h1{
+	margin:35px 0 0 0;
+	font-size:20pt;
+}
+
+br{
+	font-size:28pt;
+}
+
+h2{
+	font-size:16pt;
+}
+
+h3{
+	font-size:12pt;
+}
+
+h1,h2,h3{font-family:CENTURY GOTHIC;}
```

Comment:
Always add to font family safety font and type: font-family: "Roboto", "Arial", sans-serif;

Using multi selector, write each of them from new line:
h1,
h2,
h3 {
...
}